tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -27,20 +27,20 @@ Feature: restrict Sharing
     And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt" in the server
     And user "Brian" has logged in using the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with users in their groups
+#   shares-only-with-groups-members configuration is not available now in the oCIS webUI
```

Comment:
I didn't mention the issue link because at the top **@issue-ocis-1317** tag is present which is related to issue